deployments/aws/templates/ai-unlimited/ai-unlimited-with-alb.yaml

```diff
@@ -790,8 +906,8 @@ Resources:
       Path: /
       Roles: !If
         - NEEDSROLE
-        - [!Ref AiUnlimitedRole]
-        - [!Ref IamRoleName]
+        - - !Ref AiUnlimitedRole
+        - - !Ref IamRoleName
```

Comment:
Please ensure that ec2:DescribeVpcAttribute and ec2:DescribeAvailabilityZones are included on PolicyDocument Statements.